tests/integration/records/multi_db/test_records_table2table.py

```diff
@@ -107,9 +107,14 @@ def move_and_verify(self, source_dbname: str, target_dbname: str) -> None:
 
                 quoted_target = quote_schema_and_table(None, target_schema_name, TARGET_TABLE_NAME,
                                                     db_engine=target_engine)
-                sql = f"DROP TABLE {quoted_target}"
+                drop_target_sql = f"DROP TABLE {quoted_target}"
                 with target_conn.begin():
-                    target_conn.exec_driver_sql(sql)
+                    target_conn.exec_driver_sql(drop_target_sql)
+                quoted_source = quote_schema_and_table(None, source_schema_name, source_table_name,
+                                                       db_engine=source_engine)
+                drop_source_sql = f"DROP TABLE {quoted_source}"
+                with source_conn.begin():
+                    source_conn.exec_driver_sql(drop_source_sql)
```

Comment:
This is fixing a bug I found while I was in here:

In the table2table job, source tables were not being deleted at the end of the job and therefore thousands of them were stacking up in the long-lived dbs, i.e Redshift and BigQuery.

Reply:
amazing good job